spec/models/activity_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+RSpec.describe Activity, type: :model do
+  it { is_expected.to belong_to(:actor).inverse_of(:activities).class_name(User.name) }
+
+  let(:instance) { create(:instance) }
+  with_tenant(:instance) do
+    describe '.create' do
+      let(:user) { create(:user, name: 'tester') }
+      let(:course) { create(:course) }
+      subject(:activity_and_notifications) do
```

Comment:
@lowjoel By giving meaningful names to subject, would this help with our code readability ? 
http://betterspecs.org/#subject